mergify/README.md

```diff
@@ -87,3 +113,6 @@ Need help? Contact [Mergify support][1].
 [7]: https://github.com/DataDog/integrations-extras/blob/master/mergify/assets/service_checks.json
 [8]: https://docs.datadoghq.com/developers/integrations/new_check_howto/#configure-the-developer-tool
 [9]: https://docs.datadoghq.com/agent/guide/agent-configuration-files/#agent-configuration-directory
+[10]: https://dashboard.mergify.com
+[11]: https://app.datadoghq.com/organization-settings/api-keys?filter=Mergify
+[12]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/mergify/images/setup_integration.png
```

Comment:
This link is 404ing. It looks like you haven't added the image to the folder.

Reply:
The image is in the pull request: https://github.com/DataDog/integrations-extras/pull/2487/files#diff-efb725effa0f253201963423f0acf815ebed45752da1028745bde91688aa1a60